src/main/java/frc/robot/subsystems/RollerSubsystem.java

```diff
@@ -90,12 +87,18 @@ public RollerSubsystem() {
     }
 
     /**
-     * Opens the roller by starting the open timer.
+     * Opens the roller by starting the open timer. Does nothing if we're not allowed
+     * to open, or if opening is still on cooldown.
      */
     public void openMotor() {
+        if (!allowOpen) return;
```

Comment:
a later addition on main allowed for only a falling edge of allowOpen to trigger a close, so that the driver can still open the intake if they're deliberately doing so while underneath the allowed threshold